hook.js

```diff
@@ -61,7 +61,17 @@ function processPullRequest(destinationRepo, originRepo, originBranch, prNumber,
                     // Callback for handling an incompatible line of code
                     var line = diff.lineToIndex(file.patch, incompatibility.usage.source.start.line);
                     var comment = incompatibility.featureData.title + ' not supported by: ' + incompatibility.featureData.missing;
-                    commenter.postPullRequestComment(commentURL, comment, file.filename, currentCommit.sha, line);
+
+                    // Store a comment task in Redis for later processing
+                    redisQueue.create('comment', {
+                        commentURL: commentURL,
+                        sha: currentCommit.sha,
+                        filename: file.filename,
+                        line: line,
+                        comment: comment
+                    }).save(function(error) {
```

Comment:
Don't facepalm. :smile:

I did exactly the same thing last night and was confused for way too long about why I was seeing "null" in the log.